infra/azure/modules/batch/main.tf

```diff
@@ -74,7 +74,7 @@ resource "azurerm_storage_container" "query" {
 }
 
 resource "azurerm_storage_account" "test" {
-  name                     = "${var.resource_group.name}test"
+  name                     = "${var.az_storage_account_name}test"
```

Comment:
Good find. We definitely shouldn't dictate the name of your resource group just because we snagged the `hailtest` storage account. Can we make this name a bit more specific, more along the lines of `batch_test_user_storage_account_name`?

Reply:
This is likely also a problem for the other storage accounts (see the `batch` storage account above, we just haven't clashed on those).

Reply:
Well I'll leave the other potential clashes for now, but I'll take the name change suggestion.